server/router.js

```diff
@@ -0,0 +1,163 @@
+'use strict';
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const methodOverride = require('method-override');
+const db = require('./db');
+
+const router = express.Router();
+
+let movieCount;
+
+router.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
We're already applying this in `server/app.js` so I don't think there's any reason to use it again here.